setup.py

```diff
@@ -48,6 +48,7 @@
     "logomaker",
     "biopython",
     "tidytcells~=2.0",
+    "tqdm==4.66.4",
 ]
```

Comment:
Do we need a specific version here? It might be easier for users installing the package to not specify the version (or only a lower bound)

Reply:
Great point. I had my application hat on, not my library beret. Will change this.